src/commandExecutor.ts

```diff
@@ -108,6 +111,14 @@ const CommandExecutor = async () => {
   }
 }
 
+const printAppVersion = () => {
+  const { version, commit } = JSON.parse(fs.readFileSync(path.resolve(__dirname, '../package.json'), 'utf8'))
```

Comment:
Maybe I would move this to fileẃriter so we don't need to import `fs` here but file-writer is not the best name in that case.